_data/social.yml

```diff
@@ -277,8 +277,6 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
-      exceptions:
-          text: "SMS-capable phone required."
```

Comment:
Instead of removing this exception, we could rephrase it to say `SMS required for 2FA.` if the current exception is too vague.

Reply:
Thanks @jamcat22, I agree. @mxxcon thanks for the feedback. I did not realize this was a Facebook style SMS required scenario 😄. Will fix.

Reply:
@jadchaar Actually, Facebook 2FA  can be used without SMS #2937 :grinning: